e2e/.eslintrc.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  plugins: ['cypress'],
```

Comment:
When we remove this config here, should we add "plugin:cypress/recommended" in package.json? I guess this would include some useful default rules for cypress.

https://www.npmjs.com/package/eslint-plugin-cypress

Reply:
done with https://github.com/ecamp/ecamp3/pull/3664/commits/10f5e503d682dc707116d901fcbf033ba2e95832

(the lint check was not there before, the .eslintrc was never used)